resources/gen-ai/aws-qa-appsync-opensearch/schema.graphql

```diff
@@ -1,40 +1,63 @@
-enum ResponseGenerationMethod {
-	LONG_CONTEXT
-	RAG
+enum ModelProvider {
+	Sagemaker Endpoint
+	Bedrock
+}
+enum Modality {
+	Image
+	Text
 }
 
-type QADocs @aws_iam @aws_cognito_user_pools {
+input ModelConfiguration {
+	provider: ModelProvider
+	modelId: String
+	modality: Modality
```

Comment:
is this for input or output modalities ?

Reply:
Input modality. The construct only check for input modelid and modality compatibility.